packetbeat/protos/amqp/config.go

```diff
@@ -16,7 +16,7 @@ type amqpConfig struct {
 var (
 	defaultConfig = amqpConfig{
 		ProtocolCommon: config.ProtocolCommon{
-			TransactionTimeout: protos.DefaultTransactionTimeout,
+			TransactionTimeout: protos.DefaultTransactionExpiration,
```

Comment:
removed DefaultTransactionTimeout variable + found TransactionTimeout being used kinda inconsistently by default (some modules treat as seconds, others as milliseconds in config file)